src/library-authoring/author-library/LibraryAuthoringPage.jsx

```diff
@@ -89,9 +89,8 @@ export const BlockPreviewBase = ({
       actions={(
         <ActionRow>
           <Button
-            as={Link}
-            to={editView}
-            variant="tertiary"
+            aria-label={intl.formatMessage(messages['library.detail.block.edit'])}
+            onClick={() => setShowEditorModal(true)}
           >
             <FontAwesomeIcon icon={faEdit} className="pr-1" />
             {intl.formatMessage(messages['library.detail.block.edit'])}
```

Comment:
Should this use a FormattedMessage component?

Reply:
it's only formatting (internationalizing) the aria-label. The button itself is just an icon. (Also, we're not trying to refactor all the existing UI/UX, just replacing the older LA editor pages with an FLCC editor popup)